scripts/run-system-tests

```diff
@@ -0,0 +1,108 @@
+
+# Search string passed to `find`
+# Values: String
+SEARCH='*'
+
+# Run headed Chrome instead of headless
+# Values: anything :)
+CYPRESS_HEADED=
+
+# Don't close headed Chrome after spec finishes
+# Values: anything :)
+CYPRESS_NO_EXIT=
+
+# Maximum number of retries
+# Values: n ∈ ℕ
+RETRIES=${RETRIES:-3}
+
+# Stop if file failed $RETRIES times? (turn this off, if you want
+# to test all tests locally but something keeps failing)
+# Values: 0, 1
+STOP_ON_FAIL=${STOP_ON_FAIL:-1}
+
+# Suppresses Cypress output on failures
+# Values: 0, 1
+SUPPRESS_OUTPUT=${SUPPRESS_OUTPUT:-0}
+
+CLEANUP_TESTS=${CLEANUP_TESTS:-0}
+
+PROXY_PORT=${PROXY_PORT:-4200}
+PROXY_PID=0
+
+PROJECT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../" && pwd )"
+CYPRESS_FOLDER="results"
+TESTS_FOLDER="system-tests"
+
+PLUGINS_PATH=${PLUGINS_PATH:-''}
+
+while getopts 's:hnr:?' flag; do
+  case "${flag}" in
+    s) SEARCH=${OPTARG:-'*'} ;;
+    h) CYPRESS_HEADED="true" ;;
+    n) CYPRESS_NO_EXIT="true" ;;
+    r) RETRIES=${OPTARG} ;;
+    ?)
+      echo -e "+-------------------------------+"
+      echo -e "| DC/OS Integration test runner |"
+      echo -e "+-------------------------------+\n"
+      echo -e "Don't forget running npm run build:assets first!\n\n"
+      echo -e "Usage: \n"
+      echo -e "\tnpm run test:integration -- [-s <FILE_NAME>] [-hn] [-r <RETRIES>]\n"
+      echo -e "Options:\n"
+        printf "  %-17s %-30s \n" "-s <TestFileName>" "Run single Test File. All files if omitted"
+        printf "  %-17s %-30s \n" "-h" "Run Chrome in Headed mode"
+        printf "  %-17s %-30s \n" "-n" "Don't close Chrome after tests are finished"
+        printf "  %-17s %-30s \n" "-r <NUM_RETRIES>" "Number of retries, 3 by default"
+        printf "  %-17s %-30s \n" "-?" "Prints this help"
+        echo ""
+      exit 0 ;;
+  esac
+done
+
+# Used for datadog reporting
+REPORT=${REPORT_TO_DATADOG:-true}
+DISTRIBUTION=${REPORT_DISTRIBUTION:-oss}
+BRANCH=${REPORT_BRANCH:-$BRANCH_NAME}
+
+# System test related Variables
+CLUSTER_URL="${CLUSTER_URL:-'http://127.0.0.1:4200'}"
+CLUSTER_AUTH_TOKEN="${CLUSTER_AUTH_TOKEN:-eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJ1aWQiOiJqdWxpYW4uZ2llc2VrZUBnbWFpbC5jb20iLCJleHAiOjE1NTQxMTM3MTN9.aHq0YtEHwYVh2qt5fereXnl1ViV3SrEcKNexRxJ9oDz1eFjscLseQ1T8wfK2r5YQscT4B29hWtowOy4vBhQlyackpw-pVEW3-6hHuhS6Fz0t-oUGhvopO4zPlitSIr85tKKN1H-LslqEUhhuz9rvruEE7y9O9OzV861YidAC94vR6Lx_aigHYDfHoXm1Ing3xJSaVUft8db5kJM_P6sKXJg6nyNmcFZHYQ2hh7jLuIyUvWRfXKh0XMv_ycNQTHIqQiwkY9-1b4NSclOAPqhOjtWSYa7_xoocOkHR0CglsQzHvscQ6hZyCB3u-OREizc09X5nKXKvCc-XbpjQNLobaw}"
+CLUSTER_AUTH_INFO="${CLUSTER_AUTH_INFO:-eyJ1aWQiOiAianVsaWFuLmdpZXNla2VAZ21haWwuY29tIiwgImRlc2NyaXB0aW9uIjogIlVzZXIgYWRkZWQgdGhyb3VnaCBPSURDIElEIFRva2VuIGxvZ2luLiBJc3N1ZXI6IGh0dHBzOi8vZGNvcy5hdXRoMC5jb20vIiwgImlzX3JlbW90ZSI6IGZhbHNlfQ==}"
+INSTALLER_URL="${INSTALLER_URL:-''}"
+TEST_UUID="${TEST_UUID:-system-test}"
+
+
+# if [ -z "$(which dcos)"]; then
+#     # TODO: setup cli…
```

Comment:
not sure if it should happen in this script or as a previous one in Jekninsfile? Keep it simple